components/p3a/brave_p2a_protocols.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_P3A_BRAVE_P2A_PROTOCOLS_H_
+#define BRAVE_COMPONENTS_P3A_BRAVE_P2A_PROTOCOLS_H_
+
+#include <stdint.h>
+
+namespace brave {
+
+class DirectEncodingProtocol {
```

Comment:
you don't need a class for this, just a free function should be enough

Reply:
can certainly move this into a free function - the idea was to build out a randomisation class as we add more support but given the poc nature of this maybe premature. wdyt?

Reply:
yep, no problem then